src/marqo/core/search/recommender.py

```diff
@@ -201,13 +255,25 @@ def recommend(self,
 
         return results
 
-    def _get_default_interpolation_method(self, marqo_index: MarqoIndex) -> InterpolationMethod:
+    def get_default_interpolation_method(self, marqo_index: MarqoIndex,
+                                         context_documents: Union[List[str], Dict[str, float]]) -> InterpolationMethod:
+        """
+        Returns the default interpolation method based on the index configuration and whether context documents
+        exist. For recommend endpoint, context documents always exist.
+
+        For indexes that normalize embeddings, SLERP is used if context documents are provided, NLERP is used
+        otherwise (None).
+        """
         if marqo_index.normalize_embeddings:
-            return InterpolationMethod.SLERP
+            if context_documents is not None:
```

Comment:
What if context_documents is []

Reply:
It would still return NLERP. But in validation for both recommend and search with context docs, it will raise an InvalidArgumentError anyway.

We have:
```
        if documents is None or len(documents) == 0:
            raise InvalidArgumentError('No document IDs provided')
```
And in context docs search, we have a pydantic validator to make sure it's not empty.